.photofinish.toml

```diff
@@ -202,3 +202,13 @@ files = [
 files = [
     "./test/fixtures/scenarios/host-details/9cd46919-5f19-59aa-993e-cf3736c71053_cloud_discovery_unknown.json",
 ]
+
+[vmdrbddev01]
```

Comment:
This is not used anywhere.
I would avoid adding things that you only use locally for your tests.
If you need in reallity, just add in the corresponding PR